### PR TITLE
Issue/headless mode

### LINKED ISSUE #3: Headless mode
It would be very useful to have a headless mode where the parameters are passed to the command via flags, etc.

That would allow this to be used from other tools if required.

### PR DESCRIPTION
**Pull request by crispy101**
Closes #3

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -12,47 +12,102 @@ useradd -g ${DOCKER_GID} --home-dir /home/node -s /bin/bash -u ${DOCKER_UID} nod
...
+        case $key in
+            --profile)
+            profile="${2}"
+            shift; shift
+            ;;
+            --instances)
+            instances="${2}"
+            shift; shift
+            ;;
+            --securitygroup)
```
Comment (AdamJHall) [inline entrypoint.sh]: I'd suggest changing the cli argument flags (not the variables) to be separated by hyphens `-` e.g. `--security-group` as their just a bit more readable and maybe change `--webhookparameter` to `--slack-webhook-url` to be more inline with the others.

**Review by AdamJHall — APPROVED** [review]
Just one small comment around cli flag naming patterns, but it's just me being a little picky. Happy to approve as is, and they can be updated later if needed.

File: entrypoint.sh (hunk 1 above)
Comment (crispy101) [reply entrypoint.sh]: I love your comment! Updated some of them to be more readable.

**Review by AdamJHall — APPROVED** [review]
Looks good :D

**Review by johnjcsmith — APPROVED** [review]
:tada: Great thanks @crispy101!

**Pull request merged**